### PR TITLE
Update helm install doc

### PR DESCRIPTION
**Pull request by george-edgedelta**
Hi @baturozgur ,

I put back the text that was commented out.  Please make sure it's still technically accurate.

### CODE REVIEW

**Comment by george-edgedelta** [comment]
Hi @baturozgur 

I put back the text that was commented out. Please make sure it's still technically accurate.

File: docs/installation/helm.md [hunk 1]
```diff
@@ -79,7 +62,7 @@ secretApiKey:
   key: "ed-api-key"
```
Comment (taylanisikdemir) [inline docs/installation/helm.md:62]: fix indentation of key. it should be at same level as name

File: docs/installation/helm.md [hunk 2]
```diff
@@ -39,37 +39,20 @@ Edge Delta uses a Kubernetes-recommended, node-level log collecting architecture
...
+1. To set your **API-KEY**, you can use either **apiKey** or **secretApiKey** in the values.yml file.
 
-```
-helm install edgedelta edgedelta/edgedelta --set apiKey=<API-KEY> -n edgedelta --create-namespace
-```
-
-3. To set your **API-KEY**, you can use either **apiKey** or **secretApiKey** in the values.yml file.
-
-  - To use **apiKey** as a Kubernetes secret, change the values.yml file:
+    * To use **apiKey** as a Kubernetes secret, change the values.yml file. (Note: **apiKey** will be kept in clear text as part of your pod property.)
```
Comment (taylanisikdemir) [inline docs/installation/helm.md:49]: Update 1. and 2. as follows please:

1.To provide **API-KEY** as clear text, set **apiKey** in values.yml

```yaml
apiKey: "API-KEY"
```
2.To provide **API-KEY** as a k8s secret, set **secretApiKey** in values.yml. Note that this secret must be created before running helm.
```
secretApiKey:
  name: "ed-api-key"
  key: "value"
```

**Review by taylanisikdemir — APPROVED** [review]

File: docs/installation/helm.md (hunk 1 above)
Comment (george-edgedelta) [reply docs/installation/helm.md:62]: I don't know why the indention looks off here, but when I run a local version the site, it displays correctly
Screen Shot 2022-02-03 at 9 51 08 AM
.

**Pull request merged**